streaming/python/runtime/remote_call.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import os
+import ray
+import time
+from enum import Enum
+
+from ray.actor import ActorHandle
+from ray.streaming.generated import remote_call_pb2
+from ray.streaming.runtime.command\
+    import WorkerCommitReport, WorkerRollbackRequest
+
+logger = logging.getLogger(__name__)
+
+
+class CallResult:
+    """
+    Call Result
+    """
+
+    def __init__(self, success, result_code, result_msg, result_obj):
+        self.success = success
+        self.result_code = result_code
+        self.result_msg = result_msg
+        self.result_obj = result_obj
+
+    @staticmethod
+    def success(payload=None):
+        return CallResult(True, CallResultEnum.SUCCESS, None, payload)
+
+    @staticmethod
+    def fail(payload=None):
+        return CallResult(False, CallResultEnum.FAILED, None, payload)
+
+    @staticmethod
+    def skipped(msg=None):
+        return CallResult(True, CallResultEnum.SKIPPED, msg, None)
+
+    def is_success(self):
+        if self.result_code is CallResultEnum.SUCCESS:
+            return True
+
+        return False
+
+
+class CallResultEnum(Enum):
+    """
+    call result enum
+    """
+
+    SUCCESS = 0
+    FAILED = 1
+    SKIPPED = 2
+
+
+class RemoteCallMst:
+    """
+    remote call job master
+    """
+
+    @staticmethod
+    def request_job_worker_rollback(master: ActorHandle,
```

Comment:
why not put those methods as module function?

Reply:
I'll do this in separate PR. I don't want to change python code now, because it needs a lot of testing.